seedTestFiles/seedInsertTestUnique10mil.js

```diff
@@ -0,0 +1,89 @@
+/* eslint-disable no-await-in-loop */
+/* eslint-disable no-plusplus */
+/* eslint-disable no-console */
```

Comment:
Looks great. You can also disable certain ESLint rules in the [config](http://www.justthink.it/articles/turn-off-eslint-rule-via-config-file-via-javascript/) file, installing a [library](https://github.com/sindresorhus/eslint-plugin-unicorn/blob/master/docs/rules/no-abusive-eslint-disable.md), or specifying in a [root folder](https://stackoverflow.com/questions/42250257/disable-eslint-rules-for-folder).